steps/output.sh

```diff
@@ -21,86 +21,138 @@ echo "====================================================================="
+      );" | psqlcmd
+
+# ?? rows
+
+
+
+
+# "====================================================================="
 echo "Create table indexes"
-echo "CREATE INDEX wikipedia_article_osm_type_osm_id_idx
-      ON wikipedia_article_slim
-      (osm_type, osm_id)
-      WHERE (osm_type IS NOT NULL)
-      ;" | psqlcmd
-echo "CREATE INDEX wikipedia_article_slim_title_language_idx
-      ON wikipedia_article_slim
+# "====================================================================="
+
+echo "CREATE INDEX wikipedia_article_title_language_idx
+      ON wikipedia_article
       (title, language)
       ;" | psqlcmd
 echo "CREATE INDEX wikipedia_article_wd_page_title_idx
-      ON wikipedia_article_slim
+      ON wikipedia_article
       (wd_page_title)
       ;" | psqlcmd
 echo "CREATE INDEX wikipedia_redirect_language_from_title_idx
-      ON wikipedia_redirect_slim
+      ON wikipedia_redirect
       (language, from_title)
       ;" | psqlcmd
+echo "CREATE INDEX wikimedia_importance_title_language_idx
+      ON wikimedia_importance
+      (title, language)
+      ;" | psqlcmd
+echo "CREATE INDEX wikimedia_importance_wd_page_title_idx
+      ON wikimedia_importance
+      (wd_page_title)
+      ;" | psqlcmd
+
+
+
 
+# "====================================================================="
+echo "Dump tables"
+# "====================================================================="
 
-echo "Create wikipedia_importance.sql.gz"
+echo "* wikipedia_importance.sql.gz"
 
-pg_dump -d $DATABASE_NAME --no-owner -t wikipedia_article_slim -t wikipedia_redirect_slim | \
+pg_dump -d $DATABASE_NAME --no-owner -t wikipedia_article -t wikipedia_redirect | \
         grep -v '^SET ' | \
         grep -v 'SELECT ' | \
         grep -v '\-\- ' | \
-        sed 's/public.wikipedia_article_slim/wikipedia_article/' | \
-        sed 's/public.wikipedia_redirect_slim/wikipedia_redirect/' | \
+        sed 's/public\./' | \
         pigz -f -9 > "$OUTPUT_PATH/wikipedia_importance.sql.gz"
 
-echo "Create wikipedia_article.csv.gz"
 
 
-echo "COPY wikipedia_article_slim
-      TO '$OUTPUT_PATH_ABS/wikipedia_article.csv'
+echo "* wikipedia_article.csv.gz"
+
+rm -f "$OUTPUT_PATH_ABS/wikipedia_article.csv.gz"
+echo "COPY wikipedia_article
+      TO PROGRAM 'pigz -9 > $OUTPUT_PATH_ABS/wikipedia_article.csv'
       CSV
-      DELIMITER ','
       HEADER;" | psqlcmd
 
-pigz -f -9 "$OUTPUT_PATH_ABS/wikipedia_article.csv"
 
 
-echo "Create wikipedia_redirect.csv.gz"
+echo "* wikipedia_redirect.csv.gz"
 
-echo "COPY wikipedia_redirect_slim
-      TO '$OUTPUT_PATH_ABS/wikipedia_redirect.csv'
+rm -f "$OUTPUT_PATH_ABS/wikipedia_redirect.csv.gz"
+echo "COPY wikipedia_redirect
+      TO PROGRAM 'pigz -9 > $OUTPUT_PATH_ABS/wikipedia_redirect.csv'
       CSV
-      DELIMITER ','
       HEADER;" | psqlcmd
 
-pigz -f -9 "$OUTPUT_PATH_ABS/wikipedia_redirect.csv"
+
+
+echo "* wikimedia_importance.csv.gz"
+
+rm -f "$OUTPUT_PATH_ABS/wikimedia_importance.csv.gz"
+echo "COPY wikipedia_redirect
```

Comment:
Did you mean wikipedia_importance here?

Reply:
thanks! fixed in https://github.com/osm-search/wikipedia-wikidata/pull/47